### ISSUE TITLE
Enable GitHub Discussions

### ISSUE THREAD

**Issue by hello-smile6** [RobinLinus/snapdrop#270]
GitHub discussions would help improve contribution to the project and would enable collaboration.

**Comment by fm-sys** [comment]
Actually I don't really like the duplication between issues and discussions. Also you can't close discussions, and they are IMHO hard to read. Also I'm always struggling whether to open a issue or a discussion.

I would vote for don't enable it, but that's only my personal taste...

**Comment by Bellisario** [comment]
I love discussions, because it's the perfect way to ask and answer for some information and not for an issue... but I think that @RobinLinus needs to add a write to readme that advise the users that for questions, they need to ask them on discussions and not on issues.

**Comment by hello-smile6** [comment]
Discussions isn't enabled on this repo.

**Comment by Bellisario** [comment]
Yes, I know, in fact I'm saying that I'm for enable discussions.

**Comment by hello-smile6** [comment]
I agree, @Bellisario . If GitHub supported polls, we could do that.

**Comment by Bellisario** [comment]
We can do this @hello-smile6...

**Comment by Bellisario** [comment]
Who thinks enable GitHub reactions it's a good thing to do add a "+1" reaction and who thinks it's not really a good things to enable add "-1" reaction!

**Comment by hello-smile6** [comment]
Let's do it! We have 2 yes!

**Comment by Bellisario** [comment]
@RobinLinus, can you add discussions?
I think this is the best way to discuss with others about suggestions etc. without create an inopportune issue...
